tests/reader.tests.js

```diff
@@ -0,0 +1,202 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const { Reader, Book } = require('../src/models/index');
+const app = require('../src/app');
+
+describe('/readers', () => {
+  before(async () => {
+    await Reader.sequelize.sync({ force: true });
+  });
+
+  beforeEach(async () => {
+    await Reader.destroy({ where: {} });
+  });
+
+  describe('with no records in the database', () => {
+    describe('POST /readers', () => {
+      it('creates a new reader in the database', async () => {
+        const response = await request(app).post('/readers').send({
+          name: 'Darrow',
+          email: 'redrising@gmail.com',
+          password: 'howler123',
+        });
+        const newReaderRecord = await Reader.findByPk(response.body.id, {
+          raw: true,
+        });
+
+        expect(response.status).to.equal(201);
+        expect(response.body.name).to.equal('Darrow');
+        expect(newReaderRecord.name).to.equal('Darrow');
+        expect(newReaderRecord.email).to.equal('redrising@gmail.com');
+      });
+    });
+    it('throws an error if username is null', async () => {
```

Comment:
Just clear lines for the rest of this test, doesnt feel necessary to add a comment on all the rest of them (but please check the rest of the test files for this!):
```
    });

    it('throws an error if username is null', async () => {
```